test/controllers/sequences_controller_test.rb

```diff
@@ -287,18 +287,18 @@ def test_edit_sequence_post
     }
 
     # Prove user must be logged in to edit Sequence.
-    post(:edit, params)
+    post(:update, params)
```

Comment:
I'd like to use `patch(:update` instead of `post(:update` here and elsewhere, but can't figure out the right syntax for the arguments.

Reply:
Hi Joe -
There's a pretty good tutorial here for new-syntax update actions.
https://learn.co/lessons/rails-edit-update-action-readme
And for testing that action, you should be able to use something like patch :update, sequence: params. 
https://stackoverflow.com/questions/32310851/testing-update-action-in-minitest
If you find that's not allowed, try inherting from IntegrationControllerTest instead of FunctionalTestCase